ONSdigital/ras-frontstage
# Signed out help content

# What and why?
Updating signed out help as per Laura's recommendations https://drive.google.com/drive/u/2/folders/18T9eI0128rI54w94coqpgOwX-D5Xcs0k
Fixed breadcrumb positioning bug

# How to test?
View/read content in sign out help
Check the Breadcrumbs are positioned correctly 

# Trello
https://trello.com/c/5B5m9zGE/652-update-the-content-for-signed-out-help-on-frontstage
